modules/managing-others/Readme.md

```diff
@@ -0,0 +1,60 @@
+# About managing others
+
+> Always treat your employees exactly as you want them to treat your best customers.
+Management at Novoda is based on producing in others what you expect from yourself.
+
+The benefits Novoda gets by building a culture of engagement are: Increased Productivity, Greater Retention and Lower Costs. The benefits individuals get are: Work Satisfaction, Friendly Inspiring Colleagues, and Fair Recogniton & Compensation. You as a manager have a huge role in bringing about these goals.
+
+Management at Novoda is all about getting people to enjoy their journey through Novoda. To have enthusiasm for their work, to always be curious and to achieve something great together. 
+
+> A miserable job is different from bad one. Bad or good depends upon who you are and what you value and enjoy. A miserable job is one that you dread going to and can’t wait to leave. (Patrick M. Lencioni)
+
+To be a successul manager involves servant-leadership, encouraging others to always be better, to listen to people and point out what they are doing right whilst supporting them if they feel they got something wrong.
```

Comment:
Would split out `listening` into its own thing rather than anding it into `point out`. Something like this?

> Becoming a successul manager involves servant-leadership, encouraging others to always be better, and actively listening to people. You will need to point out what they are doing right whilst supporting them if they feel they got something wrong.